src/App.js

```diff
@@ -193,9 +194,17 @@ export const App = ({
         />
         <Route
           path='/insights'
-          render={props => (
-            <LoadableInsights {...props} isLoggedIn={isLoggedIn} />
-          )}
+          component={Loadable({
+            loader: () => import('./pages/Insights/'),
+            render: (loaded, props) => (
+              <loaded.default
+                isDesktop={isDesktop}
+                isLoggedIn={isLoggedIn}
```

Comment:
`isDesktop` could be passed here `<LoadableInsights {...props} isLoggedIn={isLoggedIn} />` as already done with the `isLoggedIn` prop + we do it like this in other places